server/rooms.ts

```diff
@@ -56,6 +56,13 @@ interface ChatRoomTable {
 	subRooms?: string[];
 }
 
+interface ShowRequest {
+	name: string;
+	link: string;
+	comment: string;
+	dimensions?: [number, number, boolean];
```

Comment:
Why isn't this just three separate properties `width`, `height`, `resized`?

Reply:
Since that's what Chat.fitImage puts out, and i felt it would unnecessarily complicate things to move it into a new object / parse it later.

Reply:
In the end, you're still going to have to destructure it, and this makes it a bit cleaner on the /requestshow side.

Reply:
Makes sense.